sig-security-assessments/cluster-api/self-assessment.md

```diff
@@ -0,0 +1,1099 @@
+as capacity planning, policy for usage of the cloud and backup and disaster
+recovery.
+
+#### Platform Operator
+
+The platform operator is focused on building a developer platform on top of
+Kubernetes. They standardise operations and create policies for the running of
+clusters on different infrastructures. They onboard development teams onto
+Kubernetes and build standardised pipelines for software delivery.
+
+#### Workload Operator
+
+The workload operator is a member of a software development team within a
+specific line of business. They may have varying levels of experience with
+Kubernetes and are interested in automation of software delivery for their LOB.
+They are also interested in monitoring, backup and restore and the consumption
+of cloud services.
+
+#### Application Developer
+
+The workload operator is a member of a software development team within a
+specific line of business. They may have little experience with Kubernetes, and
+mostly care about getting their application running. They will typically work
+only with the Kubernetes clusters they have been given access to.
+
+### Primary Components
+
+### Types of Clusters
+
+In Cluster API, we normally refer to two types of clusters: _management
+clusters_ and _workload clusters_.
+
+#### Workload Clusters
+
+Clusters managed _by_ Cluster API are called _workload clusters_, and are
+primarily what workload operators and application developers will interact with.
+
+#### Management Clusters
+
+A management cluster is a cluster that actually runs the Cluster API components,
+contains the resources representing the cluster and is reaching out to cloud
+provider services to perform infrastructure operations. We would expect the
+Cloud Admin, Platform Operator and to some extent Workload Operators to interact
+with the Management Cluster, but not typically Application Developers. The
+diagram below illustrates the relationship between management and workload
+clusters.
+
+![Schematic of relation between management cluster and workload cluster](images/mgmt-and-workload-cluster-relationship.png)
+
+**Figure 1**: Relation between management cluster and workload cluster
+
+##### Self-managed Management Cluster
+
+A self-managed management cluster is a management cluster whose lifecycle is
+being managed by Cluster API running on the management cluster itself.
+
+#### Bootstrap Cluster
+
+A bootstrap cluster is typically a Kubernetes “cluster” running on a local
+workstation - in the Cluster API book, we typically recommend the usage of kind,
+but any developer Kubernetes distribution is sufficient. This is used to spin up
+an initial cluster with Cluster API which maybe converted _into_ a management
+cluster. This is covered later. (TODO). It is expected that platform operators
+use a bootstrap cluster to provision the initial management cluster.
+
+##### Cluster API Core Controller
+
+Cluster API Core controller is a pluggable provider in Cluster API which
```

Comment:
nit; core controllers are not pluggable

Reply:
Updated and shortened to reflect this comment